hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMKeyRequest.java

```diff
@@ -485,6 +495,31 @@ protected void getFileEncryptionInfoForMpuKey(KeyArgs keyArgs,
     }
   }
 
+  /**
+   * Get parent ID for the user given keyName.
+   *
+   * @param omMetadataManager
+   * @param volumeName        - volume name.
+   * @param bucketName        - bucket name.
+   * @param keyName           - key name.
+   * @return
+   * @throws IOException
+   */
+  protected long getParentId(OMMetadataManager omMetadataManager,
```

Comment:
I think, its good to move this utility function to OmFileRequest.java class, isn't it?